line 1: make: command not found

Run make install
/runner/_work/_temp/b0f[1](https://github.com/0chain/zboxcli/actions/runs/4814450195/jobs/8572110783#step:6:1)4e5a-4b90-4960-bfbb-b7d3b1e25[8](https://github.com/0chain/zboxcli/actions/runs/4814450195/jobs/8572110783#step:6:9)7e.sh: line 1: make: command not found
Error: Process completed with exit code 127.

https://github.com/0chain/zboxcli/actions/runs/4814450195/jobs/8572110783

--- a/.github/workflows/build-zbox.yaml
+++ b/.github/workflows/build-zbox.yaml
@@ -15,10 +15,15 @@ jobs:
     runs-on: [self-hosted, build]
     steps:
 
-      - name: Setup go 1.18
+      - name: Install deps
+        run: |
+          apt update
+          apt install make -y
+
+      - name: Setup go
         uses: actions/setup-go@v2
         with:
-          go-version: '1.18' # The Go version to download (if necessary) and use.
+          go-version: '1.20.3' # The Go version to download (if necessary) and use.
 
       - name: Checkout
         uses: actions/checkout@v2
@@ -64,10 +69,10 @@ jobs:
     name: Build-windows
     runs-on: windows-latest
     steps:
-      - name: Setup go 1.18
+      - name: Setup go 1.20.3
         uses: actions/setup-go@v2
         with:
-          go-version: '1.18' # The Go version to download (if necessary) and use.
+          go-version: '1.20.3' # The Go version to download (if necessary) and use.
 
       - name: Checkout
         uses: actions/checkout@v2
@@ -113,10 +118,10 @@ jobs:
     runs-on: macos-runner-binary
     steps:
 
-      - name: Setup go 1.18
+      - name: Setup go 1.20.3
         uses: actions/setup-go@v2
         with:
-          go-version: '1.18' # The Go version to download (if necessary) and use.
+          go-version: '1.20.3' # The Go version to download (if necessary) and use.
 
       - name: Checkout
         uses: actions/checkout@v2

--- a/.github/workflows/build-zbox.yaml
+++ b/.github/workflows/build-zbox.yaml
@@ -23,7 +23,7 @@ jobs:
       - name: Setup go
         uses: actions/setup-go@v2
         with:
-          go-version: '1.20.3' # The Go version to download (if necessary) and use.
+          go-version: '1.18' # The Go version to download (if necessary) and use.
 
       - name: Checkout
         uses: actions/checkout@v2
@@ -69,10 +69,10 @@ jobs:
     name: Build-windows
     runs-on: windows-latest
     steps:
-      - name: Setup go 1.20.3
+      - name: Setup go 1.18
         uses: actions/setup-go@v2
         with:
-          go-version: '1.20.3' # The Go version to download (if necessary) and use.
+          go-version: '1.18' # The Go version to download (if necessary) and use.
 
       - name: Checkout
         uses: actions/checkout@v2
@@ -118,10 +118,10 @@ jobs:
     runs-on: macos-runner-binary
     steps:
 
-      - name: Setup go 1.20.3
+      - name: Setup go 1.18
         uses: actions/setup-go@v2
         with:
-          go-version: '1.20.3' # The Go version to download (if necessary) and use.
+          go-version: '1.18' # The Go version to download (if necessary) and use.
 
       - name: Checkout
         uses: actions/checkout@v2

--- a/.github/workflows/build-zbox.yaml
+++ b/.github/workflows/build-zbox.yaml
@@ -14,11 +14,10 @@ jobs:
 #     runs-on: ubuntu-latest
     runs-on: [self-hosted, build]
     steps:
-
       - name: Install deps
         run: |
-          apt update
-          apt install make -y
+          sudo apt update
+          sudo apt -y install build-essential nghttp2 libnghttp2-dev libssl-dev
 
       - name: Setup go
         uses: actions/setup-go@v2
@@ -27,8 +26,6 @@ jobs:
 
       - name: Checkout
         uses: actions/checkout@v2
-
-        
       
       - name: Set GITHUB_ENV
         run: |

